app/cdap/components/WrangleHome/Components/OngoingDataExploration/mock/oldData.ts

```diff
@@ -0,0 +1,297 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { ImportDatasetIcon } from '../../WrangleCard/iconStore/ImportDatasetIcon';
```

Comment:
for top level components don't use a relative import - from `'components/WragnleCard...'`

Reply:
nvm this isn't a top level one